server/jetstream_cluster.go

```diff
@@ -4749,11 +4756,6 @@ func (s *Server) jsClusteredStreamDeleteRequest(ci *ClientInfo, acc *Account, st
 		s.sendAPIErrResponse(ci, acc, subject, reply, string(rmsg), s.jsonResponse(&resp))
 		return
 	}
-	// Remove any remaining consumers as well.
-	for _, ca := range osa.consumers {
-		ca.Reply, ca.State = _EMPTY_, nil
-		cc.meta.Propose(encodeDeleteConsumerAssignment(ca))
```

Comment:
what will remove them after this code is removed? Will they be cleaned up elsewhere?

Reply:
Those assignments, the consumer assignments, are attached to the stream. So once the stream goes away those go with it. I also double checked the consumers themselves were stopped and deleted and they were.